src/app/features/auth/components/confirmation/confirmation.component.ts

```diff
@@ -1,15 +1,47 @@
-import { Component } from '@angular/core';
-import { RouterLink } from '@angular/router';
-
+import { Component, Inject, OnDestroy } from '@angular/core';
+import { Router, RouterLink } from '@angular/router';
+import { ForgotPasswordService } from '../../services/forgot-password/forgot-password.service';
+import { Notyf } from 'notyf';
+import { NOTYF } from '../../../../shared/notify/notyf.token';
+import { Subscription } from 'rxjs';
 
 @Component({
   selector: 'app-confirmation',
   standalone: true,
   imports: [RouterLink],
   templateUrl: './confirmation.component.html',
-  styleUrl: './confirmation.component.sass'
+  styleUrl: './confirmation.component.sass',
 })
-export class ConfirmationComponent {
-  constructor(){}
+export class ConfirmationComponent implements OnDestroy {
+  email: string = '';
```

Comment:
typescript already knows this is a string so there is no need to annotate the type okay